services/src/main/java/org/exoplatform/webconferencing/notification/provider/MailTemplateProvider.java

```diff
@@ -0,0 +1,60 @@
+package org.exoplatform.webconferencing.notification.provider;
```

Comment:
Header is missing

Reply:
It is not the good one. LGPL is for meeds-io organization. For exoplatform org it is AGPL
Please update